src/components/NodeLink/Legend.vue

```diff
@@ -1,4 +1,5 @@
 <script>
+/* eslint-disable vue/no-mutating-props */
```

Comment:
Why do this? Mutation of props is fundamentally a bad idea in Vue.

Reply:
It's the way the app works currently, and I didn't want to make a big modification other than linter changes in this PR. We need to refactor it so we're using best practices with emits

Reply:
Got it. Can you file an issue to fix this? It's something we can discuss at the rearchitecting meeting too.

Reply:
Opened #116